packages/design-system/.ladle/theme/drawer.css.ts

```diff
@@ -29,6 +29,9 @@ import {
 } from '../../src';
 
 export const Drawer: ThemeContext['Drawer'] = {
+  content: style({
+    overflowY: 'auto',
+  }),
```

Comment:
oh actually, i should do this in the component css, not the ladle theme. hold on.